frontend/src/pages/member/LoginPage.vue

```diff
@@ -73,12 +74,18 @@ export default {
         return;
       }
       try {
-        // TODO login API를 작성해주세요.
-        // const { email, password } = this.member;
-        // const data = await fetch("/login")
-        // TODO member 데이터를 불러와 주세요.
-        // const member = wait fetch("/members/me")
-        // this.setMember(member);
+        const { email, password } = this.member;
+        const tokenResponse = await requestPost('/login/token', {
+          email: email,
+          password: password
+        })
+        const token = tokenResponse["accessToken"];
+        window.localStorage.setItem('token', token);
+        const memberResponse = await requestGet('/members/me', {}, token);
```

Comment:
지금처럼 token으로 사용해도 되지만, 다른 token쓰는 곳 과 통일성을 위해 `window.localStorage.setItem('token', token)`를 사용해도 괜찮을 것 같아요 :) 
```suggestion
        const memberResponse = await requestGet('/members/me', {}, window.localStorage.setItem('token', token));
```

혹시 토큰 발급 요청, 로그인 정보 요청 로직을 메소드 분리 하는것은 어떻게 생각하시나요?

Reply:
수정하였습니다!!!